cmd/loadtest/main.go

```diff
@@ -71,12 +86,15 @@ func Run() int {
 		return 1
 	}
 
-	client := horizon.Client{
-		URL:  *horizonDomainFlag,
-		HTTP: &http.Client{Timeout: ClientTimeout},
-	}
+	var clients []horizon.Client
```

Comment:
needs to be a `[]*horizon.Client` since `horizon.Client` holds inner complex structs and not just primitives. apply to rest of code